### PR TITLE
Made log level corrections for failure test case logs and failure summary logs

### PR DESCRIPTION
**Pull request by rohandudam**
- Updated log_result method, changed log level of failure message
- Changed log level for failure summary and exception messages

Look at below attached screenshots:
Before log level changes:
before log level correction

After log level changes:
After log level correction

### CODE REVIEW

File: core_helpers/logging_objects.py [hunk 1]
```diff
@@ -17,20 +17,20 @@ def write_test_summary(self):
...
         if len(failure_message_list) > 0:
-            self.write('\n--------FAILURE SUMMARY--------\n')
+            self.write('\n--------FAILURE SUMMARY--------\n',level="error")
             for msg in failure_message_list:
-                self.write(msg)
+                self.write(msg,level="error")
         if len(self.exceptions) > 0:
             self.exceptions = list(set(self.exceptions))
-            self.write('\n--------USEFUL EXCEPTION--------\n')
+            self.write('\n--------USEFUL EXCEPTION--------\n',level="error")
```
Comment (shivahari) [inline core_helpers/logging_objects.py:31]: Can you we set the level to `critical` for `Exceptions`?

**Review by shivahari — CHANGES_REQUESTED** [review]

File: core_helpers/logging_objects.py [hunk 2]
```diff
@@ -60,9 +60,9 @@ def log_result(self,flag,positive,negative,level='info'):
                 self.success(negative,level="info")
         else:
             if flag is True:
-                self.success(positive,level=level)
+                self.success(positive,level="info")
```
[the remark this answers is not in the document]
Comment (rohandudam) [reply core_helpers/logging_objects.py]: I added new log level `success` and using it for `conditional_write` and `log_result` methods for `positive` messages.

File: core_helpers/logging_objects.py (hunk 1 above)
Comment (rohandudam) [reply core_helpers/logging_objects.py]: yes. I made changes.

**Review by shivahari — APPROVED** [review]
The changes look good 👍 

I ran the following checks:
- Validated that the `success` log level messages are printed in 🟢 
Success_log_level

- Validated that the `failure` log level messages are printed in 🟥 
Error_log_level

- Validated that the `critical` log level messages are printed at the end in higlighted 🟥
Exceptions_critical_log_level
 

> [!IMPORTANT]
> I was not able to complete testing the changes for the Mobile app helper module. The existing `Weather shopper` end-end test fails owing to a known issue - `pytesseract` validation

**Pull request merged**